.github/workflows/test.yml

```diff
@@ -0,0 +1,53 @@
+name: test CI
+
+on: [pull_request]
+
+jobs:
+  pytest_with_coverage:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 3
+    strategy:
+      matrix:
+        python-version: [3.8]
+    steps:
+      - name: Checkout commit
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v1
```

Comment:
Did you try setup-pytyon@v2 ?
I'm not sure difference but current implementation is v2

Reply:
V2 version's python lib still don't have pyyaml and requests built-in.